jsons/ModOptions.json

```diff
@@ -1 +1 @@
-{isBaseRuleset:true}
\ No newline at end of file
+{isBaseRuleset:true,modUrl:"https://github.com/ravignir/RekMOD",author:ravignir}
```

Comment:
Not especially necessary, those should be automatically generated when the mod is downloaded iirc

Reply:
Resolved in https://github.com/ravignir/RekMOD/pull/43/commits/998a8da7ad2d680f4a6523746e9f466095555b5d